mods/cnc/fluent/chrome.ftl

```diff
@@ -239,101 +239,111 @@ label-combat-stats-army-value-header = Army Value
 label-combat-stats-vision-header = Vision
 
 supportpowers-support-powers-palette =
-   .ready = Ready
-   .hold = On Hold
+    .ready = Ready
+    .hold = On Hold
 
 button-command-bar-attack-move =
-   .tooltip = Attack Move
-   .tooltipdesc = Selected units will move to the desired location
-    and attack any enemies they encounter en route.
+    .tooltip = Attack Move
+    .tooltipdesc =
+    Selected units will move to the desired location
+    and engage any enemies they encounter en route.
 
-    Hold <(Ctrl)> while targeting to order an Assault Move
-    that attacks any units or structures encountered en route.
+    Left-click the icon, then right-click on the target.
 
-    Left-click icon then right-click on target location.
+    Hold <(Ctrl)> while targeting to issue an Assault Move,
+    to engage any units or structures encountered en route.
 
 button-command-bar-force-move =
-   .tooltip = Force Move
-   .tooltipdesc = Selected units will move to the desired location
-     - Default activity for the target is suppressed
```

Comment:
Why was this line removed?

Reply:
It's just confusing for the end user, and it doesn't really make sense from a technical perspective as well